travis-ci/build_steps.sh

```diff
@@ -134,3 +134,17 @@ function do_build_lib {
         $BUILD_PREFIX/lib/libopenblas* \
         $BUILD_PREFIX/lib/cmake/openblas
 }
+
+function upload_to_anaconda {
+    if [ "$OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN" == "" ]; then
+        echo "OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN is not defined: skipping."
```

Comment:
This weird. The "after success" step passes but I cannot see this output in the travis log of the PR build.

Reply:
I can see it in the raw log:

```
+eval 'upload_to_anaconda '
OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN is not defined: skipping.
++upload_to_anaconda
++'[' '' == '' ']'
++echo 'OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN is not defined: skipping.'
```